docker-compose.yml

```diff
@@ -246,7 +246,7 @@ services:
   # pilot service
   openpitrix-pilot-service:
     build: .
-    image: "openpitrix:metadata"
+    image: "openpitrix"
```

Comment:
https://github.com/openpitrix/openpitrix/blob/master/deploy/kubernetes/openpitrix/metadata/openpitrix-pilot.yaml

also need to be changed